recipe/meta.yaml

```diff
@@ -1,15 +1,16 @@
-{% set version = "7.83.0" %}
+{% set version = "7.84.0" %}
 
 package:
   name: curl_split_recipe
   version: {{ version }}
 
 source:
   url: https://curl.haxx.se/download/curl-{{ version }}.tar.bz2
-  sha256: 247c7ec7521c4258e65634e529270d214fe32969971cccb72845e7aa46831f96
+  sha256: 702fb26e73190a3bd77071aa146f507b9817cc4dfce218d2ab87f00cd3bc059d
 
 build:
   number: 0
+  skip: True # [ppc64le]
```

Comment:
Why are we skipping ppc64le?

Reply:
It was originally skipped due to requiring a patch in order to build.

We now have patch added to build s390x, so I will remove this also.